types/astraled/airQualityLuminary_V0_02/uplink.js

```diff
@@ -12,218 +12,217 @@ function swap32(val) {
 }
 
 function bytesToFloat(bytes) {
-  var bits_float = bytes;
-  var sign = (bits_float >>> 31 === 0) ? 1.0 : -1.0;
-  var e = bits_float >>> 23 & 0xff;
-  var m = (e === 0) ? (bits_float & 0x7fffff) << 1 : (bits_float & 0x7fffff) | 0x800000;
-  var f = sign * m * Math.pow(2, e - 150);
+  const bitsFloat = bytes;
+  const sign = (bitsFloat >>> 31 === 0) ? 1.0 : -1.0;
+  const e = bitsFloat >>> 23 & 0xff;
+  const m = (e === 0) ? (bitsFloat & 0x7fffff) << 1 : (bitsFloat & 0x7fffff) | 0x800000;
+  const f = sign * m * Math.pow(2, e - 150);
   return f;
 }
 
-function get_byte(buff, nr) {
+function getByte(buff, nr) {
   return Bits.bitsToUnsigned(buff.substr(nr * 8, 8));
 }
 
-function get_word(buff, nr) {
+function getWord(buff, nr) {
   return swap16(Bits.bitsToUnsigned(buff.substr(nr * 8, 16)));
 }
 
-function get_long(buff, nr) {
+function getLong(buff, nr) {
   return swap32(Bits.bitsToUnsigned(buff.substr(nr * 8, 32)));
 }
 
-function get_float(buff, nr) {
-  return bytesToFloat(get_long(buff, nr));
+function getFloat(buff, nr) {
+  return bytesToFloat(getLong(buff, nr));
 }
 
 function consume(event) {
+  const {payload_hex} = event.data;
+  const buff = Bits.hexToBits(payload_hex);
 
-  var payload_hex = event.data.payload_hex;
-  var buff = Bits.hexToBits(payload_hex);
+  const dataSize = payload_hex.length / 2;
 
-  var data_size = payload_hex.length / 2;
+  const data = {};
+  const lifecycle = {};
 
-  var data = {};
-  var lifecycle = {};
-
-  var rd_pos = 0;
+  let rdPos = 0;
 
   do {
-    var para_size = get_byte(buff, rd_pos++) & 0x3F;  // get size
-    var para_type = get_byte(buff, rd_pos++);  // get type
+    const paraSize = getByte(buff, rdPos++) & 0x3F;  // get size
+    const paraType = getByte(buff, rdPos++);  // get type
 
-    switch (para_type) {
+    switch (paraType) {
       case 1:
-        lifecycle.protocol_version = "Version: " + get_byte(buff, rd_pos + 1) + "." + get_byte(buff, rd_pos);
+        lifecycle.protocolVersion = `Version: ${  getByte(buff, rdPos + 1)  }.${  getByte(buff, rdPos)}`;
         break;
       case 2:
-        lifecycle.app_version = "Version: " + get_byte(buff, rd_pos + 2) + "." + get_byte(buff, rd_pos + 1) + "p" + get_byte(buff, rd_pos);
+        lifecycle.appVersion = `Version: ${  getByte(buff, rdPos + 2)  }.${  getByte(buff, rdPos + 1)  }p${  getByte(buff, rdPos)}`;
         break;
       case 3:
-        lifecycle.lora_stack_version = "Version: " + get_byte(buff, rd_pos + 2) + "." + get_byte(buff, rd_pos + 1) + "." + get_byte(buff, rd_pos);
+        lifecycle.loraStackVersion = `Version: ${  getByte(buff, rdPos + 2)  }.${  getByte(buff, rdPos + 1)  }.${  getByte(buff, rdPos)}`;
```

Comment:
there seems to be some strange spaces in here ${  getByte(buff, rdPos + 2)  }, is this voluntary?

Reply:
this was autocorrected